### DISCUSSION TITLE
How to conditional render an element ( q-icon )

### DISCUSSION THREAD

**Discussion by Eyuel-Nigussie** [quasarframework/quasar#15451] [General - Components / Directives / etc] [answered]
I wanted to change the heart Icon when user  like a post ( just like instagram's heart icon )

I tried this using v-show but didn't work  :- 
 <q-btn
        flat
        round
        icon="favorite"
        v-show="{{ !product.liked }}"
        @click="productStore.likes(product.id)"
   />

**Comment by MilosPaunovic** [comment, marked as answer]
https://codepen.io/milospaunovic/pen/ZEMYPyN?editors=1010

**Comment by tgeber** [comment]
You can use something like this:

:icon="condition === true ? 'icon-1' : 'icon-2'" or use a function if your check is more complex.
